packages/frontend/src/components/Button/Button.tsx

```diff
@@ -30,3 +32,4 @@ const Button: FunctionComponent<ButtonProps> = ({
 };
 
 export { Button };
+export type { ButtonSizes };
```

Comment:
This causes issues with Vite hot reloading on some systems if I'm not mistaken.

We should make a Button.types.ts file then import into each consuming component separately.

Reply:
Yes. I have encountered this before.